packages/linter/src/utils/versions.ts

```diff
@@ -4,6 +4,6 @@ export const tslintVersion = '~6.1.0';
 export const tslintToEslintConfigVersion = '^2.4.0';
 export const buildAngularVersion = '~13.0.0';
 
-export const typescriptESLintVersion = '~4.33.0';
-export const eslintVersion = '7.32.0';
+export const typescriptESLintVersion = '~5.3.0';
+export const eslintVersion = '8.1.0';
```

Comment:
It is `8.2.0` in `@nrwl/linter`

Let's change this into re-exports from `@nrwl/linter/src/utils/versions`;

```ts
export {
  typescriptESLintVersion,
  tslintVersion,
  eslintVersion,
  eslintConfigPrettierVersion,
} from '@nrwl/linter/src/utils/versions';
```